addons/binding/org.openhab.binding.nibeuplink/ESH-INF/thing/f1145-channel-types.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nibeuplink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="f1145-type-40017">
+		<item-type>Number:Temperature</item-type>
+		<label>EB100-EP14-BT12 Condensor Out</label>
+		<description>Condensor out temperature, BT12</description>
+		<state pattern="%.1f °C" readOnly="true">
```

Comment:
Well, this is intentional. The API provides raw values as integers. 
So it is defined as "MetricPrefix.DECI(SIUnits.CELSIUS)" in case of the channel you quoted.

To get a readable value it needs to be scaled. Otherwise the printed value will be 220 d°C instead of 22,0 °C. In my opinion it is better to have a useful default unit. Of course people who prefer °F or Kelvin need to specify this.

The only other option would imply to drop those prefixes and implement a scaling which is independent from UoM.

Reply:
Sorry, I read this answer quite late:

I suggest an internal scaling and pass the values without metric prefix to the framework. All other bindings using UoM are doing it this way and the user has the advantage that all his values (temperature in this example) are directly presented in the correct, locale dependent unit. Otherwise users had to modify the state description on every item using this binding.

Reply:
There is also a reference for this in Eclipse SmartHome: The Yahoo Weather binding does an internal calculation on the pressure value so it can be exposed to the framework in hPa instead of kilo-inch-mercury:

https://github.com/eclipse/smarthome/blob/542bcd0869ca7e22e339f2c69f27c381329da122/extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/handler/YahooWeatherHandler.java#L227-L233

Reply:
I have changed the binding to do the scaling before applying UoM.